pbj-integration-tests/src/main/java/com/hedera/pbj/integration/fuzz/FuzzTest.java

```diff
@@ -50,7 +49,8 @@ public static <T> FuzzTestResult<T> fuzzTest(final T object, final double thresh
         }
 
         final Map<SingleFuzzTestResult, Long> resultCounts = IntStream.range(0, repeatCount)
-                .parallel()
+                // Note that we must run this stream sequentially to enable
+                // reproducing the tests for a given random seed.
```

Comment:
Running tests in parallel is very beneficial to keep test time low. In case of "fixed random", it is possible for every test to use its own `Random` instance, all built from the same seed? Would it be fully deterministic?

Reply:
I think this isn't a major concern currently:

```
$ gr fuzzTest --info
...
    Fuzz tests PASSED with passRate = 94% and deserializationFailedMean = 94%
...
> Task :fuzzTest
...
BUILD SUCCESSFUL in 47s
```

Let's solve that when that becomes a problem.

Reply:
And yeah, gr is defined as:

```
alias gr='./gradlew --rerun-tasks'
```

So it rebuilt everything related to this target, and sill managed in less than a minute.